packages/mint-components/package.json

```
@@ -56,7 +56,7 @@
     "workbox-build": "4.3.1"
   },
   "dependencies": {
-    "@saasquatch/component-boilerplate": "^1.2.2",
+    "@saasquatch/component-boilerplate": "^1.2.3-11",
```

Should use latest version?

```suggestion
    "@saasquatch/component-boilerplate": "^1.3.0",
```